modules/bloxone_anycast/variables.tf

```diff
@@ -0,0 +1,64 @@
+variable "hosts" {
+  description = "Map of hostnames with their roles, routing protocols, BGP, and OSPF configurations."
+  type = map(object({
+    role               = string
```

Comment:
if this only applies to dhcp ha group, it is best if it is marked as optional and renamed to be `ha_role`

Reply:
done